api/src/main/java/org/eclipse/microprofile/lra/annotation/Status.java

```diff
@@ -124,7 +124,7 @@
  * after it has called the Complete or Compensated method and received a response which indicates
  * that the process is in progress (with a return code 202, for example). The response code 410
  * which is received when calling this Status annotated method, MUST be interpreted by the implementation
- * that the process is successfully completed and the participant already forget about the LRA.
+ * that the process is successfully completed and the participant has already forgotten about the LRA.
```

Comment:
Why a new PR for text that is already grammatically correct.

Reply:
@mmusgrov in https://github.com/eclipse/microprofile-lra/issues/283#issuecomment-599542365 it seems that the last statement is not clear enough. I am not a native speaker but it seems to me that the wrong tense is used. @ochaloup stated there:

> the last sentence of the paragraph on the Status says that This enables the participant to free up resources.. That statement states that returning 410 or null makes participant to free up resources... right (do I misread it)?

Reply:
@mmusgrov - @ochaloup corrected me that I overlooked where exactly the unclear text was so I opened https://github.com/eclipse/microprofile-lra/pull/286. I still think that the tense here is incorrect but it's not something that I need to push. So if you think my interpretation for this fix is incorrect, go ahead and close this PR.

Reply:
Yes. There was a not well written by me at #283 what I consider as a problem. By me I think this PR may be closed. But I consider the #286 should be merged.

Reply:
> @mmusgrov - @ochaloup corrected me that I overlooked where exactly the unclear text was so I opened #286. I still think that the tense here is incorrect but it's not something that I need to push. So if you think my interpretation for this fix is incorrect, go ahead and close this PR.

@xstefank So the tense used in the sentence is correct from a grammatical stance. But I agree that your text is correct too and probably easier for a non native speakers. If the change had been part of a larger changeset then the PR would have been fine, but I declined the PR because if we allow too many PRs for these kind of stylistic changes we are going to become overloaded.

If you want to go through the whole document and make changes that would make the grammar easier to parse for non native speakers then I would support that. But bear in mind that any changes that you propose still have to make sense for native speakers too.

Reply:
I plan to do such sweep still before 1.0.